src/types/Calculator.ts

```diff
@@ -97,11 +97,27 @@ export interface ExecutionResult<T extends PopulationGroupResult> {
  */
 export interface SDEResult {
   /** Name of the SDE */
-  name: string;
+  name?: string;
   /** Raw result of SDE clause */
   rawResult?: any;
   /** Pretty result for this SDE. */
   pretty?: string;
+  /** The id of the SDE as defined in the measure. */
+  id?: string;
+  /** Criteria expression for this SDE. */
+  criteriaExpression?: string;
+  /** Measure data usage for this SDE. */
+  usage: SupplementalDataUsage;
```

Comment:
Just based on the task description, it looks like this was intended to be optional. Did that change?

Reply:
yah- double checked with Matt on it this morning!